docker/modules/Dockerfile.faux1

```diff
@@ -34,12 +34,12 @@ FROM daqf/aardvark:latest
 RUN $AG update && $AG install openjdk-8-jre
 RUN $AG update && $AG install openjdk-8-jdk git
 RUN $AG update && $AG install isc-dhcp-client ethtool network-manager netcat curl\
-    python ifupdown openssl nano apache2-utils ntpdate 
+    python ifupdown openssl nano apache2-utils ntpdate vzctl
 
 # Additional OS dependencies
 RUN $AG update && $AG install -y telnetd && $AG install xinetd nginx
 
-COPY subset/ntp/NTPClient NTPClient
+COPY subset/network/NTPClient NTPClient
```

Comment:
this shouldn't still be in subset/ -- it should be moved to docker/include/network

there should be no dependency on the subset/ folder from faux

Reply:
@grafnu I was planning to address the dependancy as a separate PR as it involved other test modules too. I'll update to pick up the network module changes here